torch_xla/distributed/cluster.py

```diff
@@ -336,22 +337,22 @@ def add_client_worker(request_id, resp, exception):
 
     return workers
 
-  def get_service_workers(self):
```

Comment:
We have tests for `cluster.py` in https://github.com/pytorch/xla/blob/master/test/test_xla_dist.py could you fix those as those are failing in CI?

Reply:
will update and verify